File: client/src/Launcher.java
import java.util.Optional;

import controller.Game;
import model.game.Map;
import model.game.MapGenerator;
import view.Assets;
import view.Gui;

/**
 * @author Julian Schelker
 */
public class Launcher {
	public static void main(String[] args) {
		System.setProperty("sun.java2d.opengl", "True");
		System.setProperty("sun.java2d.accthreshold", "0");

		// Log.setLevel(Log.Level.INFO);

		Launcher launcher = new Launcher();
		launcher.init();
	}

	void init() {
		Assets.load();
		Map map = MapGenerator.getMap();
		Optional<Runnable> shutdown = Optional.of(() -> System.exit(0));
		Gui gui = new Gui(shutdown);

		Game game = new Game(map, gui);
		Assets.waitUntilAssetsAreLoaded();
		game.start();
	}
}

File: client/src/Launcher.kt
import ch.judos.generic.control.Log
import core.base.Controller
import core.input.InputController
import core.window.GameWindow
import core.window.ResizableWindow
import scene.menu.MenuScene


class Launcher {
	fun init() {
		val input = InputController()
		//		GameWindow window = new NativeFullscreen(input);
		val window: GameWindow = ResizableWindow("TeamSnakes", 1650, 1040, input)
		val controller = Controller(window)
		controller.start()
		controller.loadScene(MenuScene::class.java)
	}
}

fun main() {
	Log.getInstance().logToFile = false
	Log.getInstance().currentLogLevel = Log.Level.INFO
	System.setProperty("sun.java2d.opengl", "True")
	System.setProperty("sun.java2d.accthreshold", "0")
	val launcher = Launcher()
	launcher.init()
}
